tests/accumulator_test.py

```diff
@@ -129,14 +129,34 @@ def test_merge_accumulators(self):
 
         self.assertEqual(merged_accumulator.compute_metrics(), 10)
 
+        vec_sum_accumulator1 = accumulator.VectorSummationAccumulator(
+            params=None,
+            values=[(15, 2)])
+        vec_sum_accumulator2 = accumulator.VectorSummationAccumulator(
+            params=None,
+            values=[(27, 40)])
```

Comment:
Please use np.array in some of the input, to test that np.arrays are processed correctly

Reply:
Done.